src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -808,6 +808,7 @@ void _model_PropertyChanged(object sender, PropertyChangedEventArgs e)
                         this.PublishCurrentWorkspaceCommand.RaiseCanExecuteChanged();
                     RaisePropertyChanged("IsPanning");
                     RaisePropertyChanged("IsOrbiting");
+                    SearchViewModel.OnWorkspaceChanged(sender,e);
```

Comment:
I think this is not required, `SearchViewModel` has a reference to `DynamoViewModel`, so `SearchViewModel` should subscribe to `DynamoViewModel.PropertyChanged` event and check for `e.PropertyName = "CurrentSpace"`.